caf-audit-binding-elasticsearch/src/test/java/com/hpe/caf/auditing/elastic/ElasticAuditIT.java

```diff
@@ -125,10 +131,13 @@ public void testIncorrectESPort() throws Exception
             //  No need to set up custom data as we expect the call to index the document to fail because
             //  of the unexpected port used.
             auditEventBuilder.send();
+        } catch (final ConnectException ex) {
```

Comment:
Why are we doing this instead of using `Assertions.assertThrows()` like you are elsewhere?

Reply:
Here the exception happens inside the try (..){}.
![image](https://github.com/CAFAudit/audit-service/assets/11929432/2d388187-ce05-4851-a6cf-4972ce337c1d).

So, couldn't use assertThrows().